### ISSUE TITLE
f5-sdk issues with creating VxLAN profile

### ISSUE THREAD

**Issue by jgruber** [F5Networks/f5-openstack-agent#196]
#### OpenStack Release
Mitaka

#### Description
Issue with agent's use of f5-sdk with the version installed through pip for f5-openstack-agent.

network_helper.py line 69

```
2016-07-14 19:24:09.448 14820 DEBUG f5_openstack_agent.lbaasv2.drivers.bigip.network_helper [req-aa8b2b22-7527-400f-ab44-eac29a8df00f - - - - -] f5_openstack_agent.lbaasv2.drivers.bigip.network_helper.NetworkHelper method create_vxlan_multipoint_profile called with arguments (<f5.bigip.ManagementRoot object at 0x7fa95bec7850>, 'vxlan_ovs') {'partition': 'Common'} wrapper /opt/stack/venv/neutron-20160503T082438Z/lib/python2.7/site-packages/oslo_log/helpers.py:45
2016-07-14 19:24:09.448 14820 CRITICAL neutron [req-aa8b2b22-7527-400f-ab44-eac29a8df00f - - - - -] AttributeError: '<class 'f5.bigip.tm.net.Net'>' object has no attribute 'tunnels_s'
2016-07-14 19:24:09.448 14820 ERROR neutron Traceback (most recent call last):
2016-07-14 19:24:09.448 14820 ERROR neutron   File "/opt/stack/service/neutron/venv/bin/f5-oslbaasv2-agent", line 11, in <module>
2016-07-14 19:24:09.448 14820 ERROR neutron     sys.exit(main())
2016-07-14 19:24:09.448 14820 ERROR neutron   File "/opt/stack/venv/neutron-20160503T082438Z/lib/python2.7/site-packages/f5_openstack_agent/lbaasv2/drivers/bigip/agent.py", line 68, in main
2016-07-14 19:24:09.448 14820 ERROR neutron     mgr = manager.LbaasAgentManager(cfg.CONF)
2016-07-14 19:24:09.448 14820 ERROR neutron   File "/opt/stack/venv/neutron-20160503T082438Z/lib/python2.7/site-packages/f5_openstack_agent/lbaasv2/drivers/bigip/agent_manager.py", line 231, in __init__
2016-07-14 19:24:09.448 14820 ERROR neutron     self._load_driver(conf)
2016-07-14 19:24:09.448 14820 ERROR neutron   File "/opt/stack/venv/neutron-20160503T082438Z/lib/python2.7/site-packages/f5_openstack_agent/lbaasv2/drivers/bigip/agent_manager.py", line 279, in _load_driver
2016-07-14 19:24:09.448 14820 ERROR neutron     self.conf)
2016-07-14 19:24:09.448 14820 ERROR neutron   File "/opt/stack/venv/neutron-20160503T082438Z/lib/python2.7/site-packages/oslo_utils/importutils.py", line 44, in import_object
2016-07-14 19:24:09.448 14820 ERROR neutron     return import_class(import_str)(*args, **kwargs)
2016-07-14 19:24:09.448 14820 ERROR neutron   File "/opt/stack/venv/neutron-20160503T082438Z/lib/python2.7/site-packages/f5_openstack_agent/lbaasv2/drivers/bigip/icontrol_driver.py", line 356, in __init__
2016-07-14 19:24:09.448 14820 ERROR neutron     self.connect_bigips()
2016-07-14 19:24:09.448 14820 ERROR neutron   File "/opt/stack/venv/neutron-20160503T082438Z/lib/python2.7/site-packages/f5_openstack_agent/lbaasv2/drivers/bigip/icontrol_driver.py", line 368, in connect_bigips
2016-07-14 19:24:09.448 14820 ERROR neutron     local_ips = self.network_builder.initialize_tunneling()
2016-07-14 19:24:09.448 14820 ERROR neutron   File "/opt/stack/venv/neutron-20160503T082438Z/lib/python2.7/site-packages/f5_openstack_agent/lbaasv2/drivers/bigip/network_service.py", line 90, in initialize_tunneling
2016-07-14 19:24:09.448 14820 ERROR neutron     partition='Common')
2016-07-14 19:24:09.448 14820 ERROR neutron   File "/opt/stack/venv/neutron-20160503T082438Z/lib/python2.7/site-packages/oslo_log/helpers.py", line 46, in wrapper
2016-07-14 19:24:09.448 14820 ERROR neutron     return method(*args, **kwargs)
2016-07-14 19:24:09.448 14820 ERROR neutron   File "/opt/stack/venv/neutron-20160503T082438Z/lib/python2.7/site-packages/f5_openstack_agent/lbaasv2/drivers/bigip/network_helper.py", line 68, in create_vxlan_multipoint_profile
2016-07-14 19:24:09.448 14820 ERROR neutron     p = bigip.tm.net.tunnels_s.vxlans.vxlan
2016-07-14 19:24:09.448 14820 ERROR neutron   File "/opt/stack/venv/neutron-20160503T082438Z/lib/python2.7/site-packages/f5/bigip/mixins.py", line 100, in __getattr__
2016-07-14 19:24:09.448 14820 ERROR neutron     raise AttributeError(error_message)
2016-07-14 19:24:09.448 14820 ERROR neutron AttributeError: '<class 'f5.bigip.tm.net.Net'>' object has no attribute 'tunnels_s'
```

#### Agent Version
9.0.1

#### Operating System
Ubuntu 14.04
RHEL 7.1

**Comment by swormke** [comment]
@jgruber  we are going to close this issue, there is a fix for this in PR #193

**Comment by zancas** [comment]
Confirmed this has been fixed in the sdk, the fix is in version 1.0.2, the `liberty` branch of this repo specifies `f5sdk==1.0.2` in `setup.py`.
